services/src/test/scala/cromwell/services/womtool/DescriberSpec.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.services.womtool
+
+import better.files.File
+import cromwell.core.WorkflowSourceFilesCollection
+import cromwell.languages.config.{CromwellLanguages, LanguageConfiguration}
+import cromwell.services.womtool.WomtoolServiceMessages.DescribeSuccess
+import io.circe.Json
+import org.scalatest.{Assertion, FlatSpec, Matchers}
+
+import scala.util.Try
+
+class DescriberSpec extends FlatSpec with Matchers {
```

Comment:
The WaaS WDLs have a laser focus on super-complicated input and output types to test the JSON generator, so they don't overlap too much with the womtool cases

Also, the directory structure differs because WaaS signals failure "in-band" with `"valid": false` and does not have separate valid/invalid directories

I felt like trying to create a superset of both would save a a bit of duplication but make things harder to understand

Reply:
I see these as very similar to the `inputs` tests (rather than the `validate` tests) - ie we could store the JSON we expect to see, and reference files from the `validate/valid` dirs (which doubles as the single "place we store WDLs to test womtool against"). 

FWIW I imagine that there is a fair amount of overlap between "WDLs of interest to `WomtoolInputsSpec`" and "WDLs of interest to `WomtoolDescribeSpec`"

Reply:
I am increasingly convinced and have started investigating how to pull this off, yet I see you just thumbed

Does that mean you're cool deferring this change to the next PR in this area?

Reply:
A leading factor that convinces me is that this is a step that moves us closer to having womtool and WaaS depend on a shared, extracted bunch of code